tools/scylla-nodetool.cc

```diff
@@ -3342,6 +3346,18 @@ For more information, see: https://opensource.docs.scylladb.com/stable/operating
             },
             repair_operation
         },
+        {
+            {
+                "resetlocalschema",
+                "Reset node's local schema and resync",
+R"(
+For more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/resetlocalschema.html
```

Comment:
Is that page supposed to be added with this PR? Otherwise, right now, it's 404.

Reply:
We have this page, but only in master, not yet in stable: https://opensource.docs.scylladb.com/master/operating-scylla/nodetool-commands/resetlocalschema.html
See https://github.com/scylladb/scylladb/issues/18276.

Reply:
works on 'master': https://opensource.docs.scylladb.com/master/operating-scylla/nodetool-commands/resetlocalschema.html
I think it was added recently - should be backported

Reply:
@mykaul that page was added in dfa88ccc282477d7de14861a6278590ec378e055. guess we have not updated the "stable" build of the webpage. but presumably, we will once 5.5 is out.